packs/data/spell-effects.db/spell-effect-dread-aura.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "ubHqpJwUwygkc2dR",
+    "img": "systems/pf2e/icons/spells/dread-aura.webp",
+    "name": "Spell Effect: Dread Aura",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Dread Aura]{Dread Aura}</em></p>\n<p>Foes in the area are frightened 1 and unable to reduce the condition.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 4
+        },
+        "rules": [
+            {
+                "key": "GrantItem",
+                "onDeleteActions": {
+                    "grantee": "cascade",
+                    "granter": "cascade"
```

Comment:
Do you need cascade on both?

Reply:
Good question. I suppose it could be just "granter", but this is how we handled both "Effect: Despair" and "Effect: Utter Despair". Should they all be "granter"?

Reply:
Test it out I guess.